### ISSUE TITLE
When a story validation fails, the message should include the story name

### ISSUE THREAD

**Issue by strika** [jonspalmer/view_component_storybook#45]
In this case, a component story had a control for a parameter that was removed from the component. Story validations correctly showed that a parameter doesn't exist, but the story name was not included in the message:

```
project|main > bin/rails view_component_storybook:write_stories_json
Writing Stories JSON
Basic::AlertMessage::ComponentStories => /home/strika/project/spec/components/stories/basic/alert_message/component.stories.json
Basic::AlternativeAlertMessage::ComponentStories => /home/strika/project/spec/components/stories/basic/alternative_alert_message/component.stories.json
Basic::Avatar::ComponentStories => /home/strika/project/spec/components/stories/basic/avatar/component.stories.json
Basic::BetaAlert::ComponentStories => /home/strika/project/spec/components/stories/basic/beta_alert/component.stories.json
Basic::BugLabel::ComponentStories => /home/strika/project/spec/components/stories/basic/bug_label/component.stories.json
Basic::FlashMessage::ComponentStories => /home/strika/project/spec/components/stories/basic/flash_message/component.stories.json
Basic::GroupLabel::ComponentStories => /home/strika/project/spec/components/stories/basic/group_label/component.stories.json
Basic::Label::ComponentStories => /home/strika/project/spec/components/stories/basic/label/component.stories.json
rails aborted!
ActiveModel::ValidationError: Validation failed: Param is not included in the list
/home/strika/project/.bundle/ruby/2.6.0/gems/activemodel-6.0.4/lib/active_model/validations.rb:411:in `raise_validation_error'
/home/strika/project/.bundle/ruby/2.6.0/gems/activemodel-6.0.4/lib/active_model/validations.rb:383:in `validate!'
/home/strika/project/.bundle/ruby/2.6.0/gems/view_component_storybook-0.8.0/lib/view_component/storybook/controls/control_config.rb:22:in `to_csf_params'
/home/strika/project/.bundle/ruby/2.6.0/gems/view_component_storybook-0.8.0/lib/view_component/storybook/story_config.rb:23:in `block in to_csf_params'
/home/strika/project/.bundle/ruby/2.6.0/gems/view_component_storybook-0.8.0/lib/view_component/storybook/story_config.rb:22:in `each'
/home/strika/project/.bundle/ruby/2.6.0/gems/view_component_storybook-0.8.0/lib/view_component/storybook/story_config.rb:22:in `to_csf_params'
/home/strika/project/.bundle/ruby/2.6.0/gems/view_component_storybook-0.8.0/lib/view_component/storybook/stories.rb:31:in `map'
/home/strika/project/.bundle/ruby/2.6.0/gems/view_component_storybook-0.8.0/lib/view_component/storybook/stories.rb:31:in `to_csf_params'
/home/strika/project/.bundle/ruby/2.6.0/gems/view_component_storybook-0.8.0/lib/view_component/storybook/stories.rb:38:in `block in write_csf_json'
/home/strika/project/.bundle/ruby/2.6.0/gems/view_component_storybook-0.8.0/lib/view_component/storybook/stories.rb:37:in `open'
/home/strika/project/.bundle/ruby/2.6.0/gems/view_component_storybook-0.8.0/lib/view_component/storybook/stories.rb:37:in `write_csf_json'
/home/strika/project/.bundle/ruby/2.6.0/gems/view_component_storybook-0.8.0/lib/view_component/storybook/tasks/write_stories_json.rake:8:in `block (3 levels) in <main>'
/home/strika/project/.bundle/ruby/2.6.0/gems/view_component_storybook-0.8.0/lib/view_component/storybook/tasks/write_stories_json.rake:7:in `each'
/home/strika/project/.bundle/ruby/2.6.0/gems/view_component_storybook-0.8.0/lib/view_component/storybook/tasks/write_stories_json.rake:7:in `block (2 levels) in <main>'
/home/strika/project/.bundle/ruby/2.6.0/gems/railties-6.0.4/lib/rails/commands/rake/rake_command.rb:23:in `block in perform'
/home/strika/project/.bundle/ruby/2.6.0/gems/railties-6.0.4/lib/rails/commands/rake/rake_command.rb:20:in `perform'
/home/strika/project/.bundle/ruby/2.6.0/gems/railties-6.0.4/lib/rails/command.rb:48:in `invoke'
/home/strika/project/.bundle/ruby/2.6.0/gems/railties-6.0.4/lib/rails/commands.rb:18:in `<main>'
/home/strika/project/.bundle/ruby/2.6.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/home/strika/project/.bundle/ruby/2.6.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/home/strika/project/.bundle/ruby/2.6.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/home/strika/project/.bundle/ruby/2.6.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/home/strika/project/.bundle/ruby/2.6.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/home/strika/project/.bundle/ruby/2.6.0/gems/activesupport-6.0.4/lib/active_support/dependencies.rb:324:in `block in require'
/home/strika/project/.bundle/ruby/2.6.0/gems/activesupport-6.0.4/lib/active_support/dependencies.rb:291:in `load_dependency'
/home/strika/project/.bundle/ruby/2.6.0/gems/activesupport-6.0.4/lib/active_support/dependencies.rb:324:in `require'
/home/strika/project/bin/rails:9:in `<top (required)>'
/home/strika/project/.bundle/ruby/2.6.0/gems/spring-1.7.1/lib/spring/client/rails.rb:28:in `load'
/home/strika/project/.bundle/ruby/2.6.0/gems/spring-1.7.1/lib/spring/client/rails.rb:28:in `call'
/home/strika/project/.bundle/ruby/2.6.0/gems/spring-1.7.1/lib/spring/client/command.rb:7:in `call'
/home/strika/project/.bundle/ruby/2.6.0/gems/spring-1.7.1/lib/spring/client.rb:30:in `run'
/home/strika/project/.bundle/ruby/2.6.0/gems/spring-1.7.1/bin/spring:49:in `<top (required)>'
/home/strika/project/.bundle/ruby/2.6.0/gems/spring-1.7.1/lib/spring/binstub.rb:11:in `load'
/home/strika/project/.bundle/ruby/2.6.0/gems/spring-1.7.1/lib/spring/binstub.rb:11:in `<top (required)>'
/home/strika/project/bin/spring:13:in `<top (required)>'
bin/rails:3:in `load'
bin/rails:3:in `<main>'
Tasks: TOP => view_component_storybook:write_stories_json
(See full trace by running task with --trace)
```

Even though it's sometimes easy to conclude which story isn't valid, it would be useful to make that apparent. What do you think? Does this make sense?

Another usecase where this can be especially useful is when `view_component_storybook:write_stories_json` is executed as part of a CI pipeline.

**Comment by hjhart** [comment]
I've been having the same issue. After using it for a while, I know now that it's the next story, alphabetically, where the problem is. Also the error message is a bit cryptic. 

I think it wouldn't be too difficult to get better error messaging around this! +1 for this feature.

**Comment by jonspalmer** [comment]
closed by #49
